doc/source/examples/dem/bouncing-particle/bouncing-particle.rst

```diff
@@ -123,14 +123,14 @@ Once the 6 simulations have been run, used the following line in your command li
 A figure will then appear comparing the expected values results with the analytical solution.
 
 Results and discussion
----------
+-----------------------
 Animation of a bouncing particle with different restitution coefficient (:math:`K_n` = 5E6 N/m):
 
 .. raw:: html
 
     <iframe width="560" height="315" src="https://www.youtube.com/embed/B1n776jHdoo" frameborder="0" allowfullscreen></iframe>
 
-It can be seen that the particle with a restitution coefficient of 1.0 doesn't have any change in its rebounds. The other particles show a reduction in rebound height that seems to respect the analytical solution express earlier.
+It can be seen that the particle with a restitution coefficient (:math:`e`) of 1.0 doesn't have any change in its rebounds. The other particles show a reduction in rebound height that seems to respect the analytical solution express earlier.
```

Comment:
```suggestion
It can be seen that the particle with a restitution coefficient (:math:`e`) of 1.0 always rebounds at the same height. The other particles show a reduction in rebound height which follows the analytical solution expressed earlier.
```